crates/manifest/src/lib.rs

```diff
@@ -306,6 +396,8 @@ pub enum TriggerConfig {
     Http(HttpConfig),
     /// Redis trigger configuration
     Redis(RedisConfig),
+    /// External trigger configuration
+    External(HashMap<String, String>),
```

Comment:
Maybe use `toml::Value` for the value here to allow non-string values? I _think_ it should work fine with `ValuesMapBuilder::serializable` (instead of `::string`) below.

Reply:
Ooh nice!  Should I do this for the application-level trigger info as well?  That's currently:

```rust
pub struct ExternalTriggerConfiguration {
    trigger_type: String,
    parameters: HashMap<String, String>,
}
```

Reply:
Tried changing it and it seemed to work so YOLO